CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
cc @danielgrant what's the intention here?

Reply:
This is to comply with the open source RFC - https://github.com/digirati-co-uk/rfc/tree/master/rfc/rfc2---open-source-standards#contributing

Reply:
Yep, this is more one for @danielgrant about why this is in there in the first place.

Reply:
It's part of an effort to make projects more appealing and inviting to contributors, and also forms part of GitHub's Open Source Guide (https://opensource.guide/). You can view project alignment to this guidance under "Insights" -> "Community" (i.e. https://github.com/dlcs/digirati-taxonomy-manager/community).

That being said, I'm not averse to removing it if someone can make a case for why it shouldn't be in there.